Dockerfile

```diff
@@ -11,7 +11,11 @@ RUN pip install --upgrade pip
 
 WORKDIR /builder
 
-COPY src/backend/marsha src/backend/setup.* /builder/
```

Comment:
Do you mean that `/builder/marsha` wasn't created with his content ?

Reply:
Yep, this is like : `COPY src/backend/marsha/* /builder/` after that, the `/builder/` directory contains the content of the `marsha` folder, the folder itself is not preserved, you end with a `/builder/bbb/` folder, a `/builder/core/`, ...

Reply:
I'm surprised about `COPY src/backend/marsha/* /builder/` :open_mouth: 
For your main concern, lunika may have answers.

Reply:
Without the fix:

``` bash
# docker build --target back-builder -f Dockerfile -t base_back-builder .
# docker run -ti base_back-builder:latest bash                           
root@bc3401d11add:/builder# ls /builder/
__init__.py  asgi.py  bbb  build  core	development  e2e  markdown  marsha.egg-info  settings.py  setup.cfg  setup.py  static  urls.py	websocket  workers.py
```


``` bash
# docker -v                                                              
Docker version 20.10.7, build 20.10.7-0ubuntu5.1
```

Reply:
I wasn't asking for a proof :sweat_smile: 
It's the difference between docker `copy` and bash `cp` that surprised me :)